test/integration/lib.rb

```diff
@@ -0,0 +1,125 @@
+require 'fileutils'
+require 'open3'
+require 'tmpdir'
+require 'test/unit'
+
+module TestLib
+  TMP_DIR = File.join(Dir.tmpdir, 'cpm-test', Time.now.strftime('%Y_%m_%d-%H_%M_%S'))
```

Comment:
We could. It's in your user's temp directory, so it it subject to deletion by the OS after some time. Also we can't delete it in case of failing tests. There will likely be important stuff inside which can help you track the error

Reply:
Actually, on second thought, this wont be that easy. The deletion can be added at the end of `runner.rb` (and skipped if tests have failed), but then the directory won't be deleted if one starts individual tests (a feature that I'd like to keep)

Reply:
Ok gotcha, just wondering if it might be better to store them in a project specific directory (analogous to a build directory) as opposed to a system one, since the growing integration test suite may acquire quite a bit of data per run. That way it would be easier to remove the tmp directory after running the test suite.